src-built-in/components/appLauncher2/src/components/AppDefinition.jsx

```diff
@@ -0,0 +1,26 @@
+import './AppDefinition.css'
+
+import React from  'react'
+
+export default class AppDefinition extends React.Component {
+
+	constructor(props) {
+		super(props)
+	}
+
+	render() {
+		const app = this.props.app
+		return (
+			<div className="app-item">
+				<div className="app-item-title">{app.friendlyName}</div>
+					<div className="app-item-tags">
+						{
+							app.tags.map((tag, index) => {
+								return <span key={index}>{tag}, </span>
```

Comment:
add a check here. `if(index !== app.tags.length)`, then render the `,`.

Reply:
Thanks for pointing to this, did it slightly different 

> {app.tags[index+1] ? `${tag}, ` : `${tag}`}